src/views/List.jsx

```diff
@@ -77,12 +77,18 @@ export function List({ data, listToken, loading }) {
 
 	console.log('filteredGroups', filteredGroups);
 	// get information by index
+
 	return (
 		<div className="list-container">
 			{loading ? (
 				<p>Your list is loading...</p>
 			) : (
 				<>
+					<button onClick={logOut}>Log Out</button>
+					<p>
+						Your list name is{' '}
+						<span style={{ color: 'salmon' }}>{listToken}</span>.
```

Comment:
I really like rendering this a different color! Good thinking and easy way to make the token easy to spot.

Reply:
✨  thank you! ✨